schema/schema.go

```diff
@@ -50,6 +51,16 @@ type Column struct {
 	Ignored Ignored
 }
 
+// Fkey respresents a foreign key.
```

Comment:
I think some of these fields need some explanation.
In particular it isn't obvious what the strings for OnDelete and OnUpdate should be.
Also, maybe worth a comment that Name is the name of the foreign key constraint -- yeah, maybe that's obvious.

Reply:
How about expanding the comment for ForeignKey:

// ForeignKey represents a foreign key.
// Note that the fields onDelete and onUpdate describe actions
// for when keys are deleted or updated. Different source databases
// support different actions. For example, mysql supports RESTRICT,
// CASCADE, SET NULL, NO ACTION, and SET DEFAULT
// (see https://dev.mysql.com/doc/refman/5.6/en/create-table-foreign-keys.html).

Reply:
updated the comment